partcad/src/partcad/part_factory_sweep.py

```
@@ -0,0 +1,178 @@
+                while exp.More():
+                    face = exp.Current()
+                    maker = BRepOffsetAPI_MakePipe(self.axis_approx, face)
+                    maker.Build()
+                    shape = maker.Shape()
+                    builder.Add(compound, shape)
+                    exp.Next()
+
+                shape = compound
+
+            except Exception as e:
+                pc_logging.exception("Failed to create a swept part: %s" % e)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)